htdocs/societe/class/societe.class.php

```diff
@@ -5276,4 +5276,98 @@ public function getKanbanView($option = '', $arraydata = null)
 		$return .= '</div>';
 		return $return;
 	}
+	
+	/**
+	 *    Get array of all contacts for a society (stored in societe_contacts instead of element_contacts for all other objects)
+	 *
+	 *    @param	int			socid		id of societe
+	 *    @param	int         $list       0:Return array contains all properties, 1:Return array contains just id
+	 *    @param    string      $code       Filter on this code of contact type ('SHIPPING', 'BILLING', ...)
+	 *	  @param    string      $element       Filter on this element of default contact type ('facture', 'propal', 'commande' ...)
+	 *    @return	array|int		        Array of contacts, -1 if error
+	 * 
+	 *
+	 * 
+	 */
+	function liste_contact_societe($list = 0, $code = '', $element = '')
```

Comment:
When we introduce a new method, we should use camel names.
I suggest:
getContacts(...)

Reply:
Ok venerated Yoda !
I change method's name, buy don't have right to change label ...